SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -3734,6 +3734,60 @@ \subsection{Test Return Value}
 allowed in ITK.
 
 
+\section{Writing Examples}
+\label{sec:WritingExamples}
+
+Many ITK examples are used in this software guide to demostrate ITK's
+architecture and development.
+
+Thanks to scripting work, parts of the \code{*.cxx} example files within
+special placeholders are included in this software guide. The \LaTeX
+placeholders available to the code for such purpose are:
+
+\begin{itemize}
+\item \textbf{Software Guide : BeginLatex} and
+\textbf{Software Guide : EndLatex}: the text within these placeholders is
+included in this software guide for text explanations, e.g.
```

Comment:
We need to add `\end{itemize}` here :bug:

Reply:
:blush: My bad. I missed paying attention to the CDash build. sorry. 84f9b47 should fix this.